src/clients/SparkCore.js

```diff
@@ -733,9 +745,9 @@ class SparkCore extends EventEmitter {
     data: Buffer,
     ..._:void[]
   ): Promise<*> => {
-    /*TODO: data type! */
+    // TODO: data type!
```

Comment:
I refactored a bunch of functions and removed parameters.  This makes it so if you call a function with extra parameters, flow will get mad.